### ISSUE TITLE
Attempted to assign to readonly property

### ISSUE THREAD

**Issue by marcshilling** [facebook/react#6277]
Getting a crash report from a React Native app that's crashing in react code. I can't really provide much detail other than the stack trace. All I know is it's our most frequent crash by far (73% of crashes are this one). Has anyone else run into this?

```
*** Terminating app due to uncaught exception 'RCTFatalException: Unhandled JS Exception: Attempted to assign to readonly property.', reason: 'Unhandled JS Exception: Attempted to assign to readonly property., stack:
m@17:659
recordTouchTrack@17:1373
extractEvents@16:30232
extractEvents@16:8354
handleTopLevel@17:4047
_receiveRootNodeIDEvent@17:3437
receiveTouches@17:3819
value@12:23605
<unknown>@12:21824
_@12:20918
value@12:21796
```

**Comment by jimfb** [comment]
This repository tracks the React core, and react-web.  The react native repository is at https://github.com/facebook/react-native, so you could try posting there if you believe this is a bug in React native.  If this is a usage question related to your own code, StackOverflow would be the appropriate place.  Regardless, I'm going to close this out, since this is not the correct repository.

**Comment by marcshilling** [comment]
I found the bug...it's in the react core code base...sooooo??

**Comment by jimfb** [comment]
... can you elaborate?  How do we reproduce this in the core?

**Comment by marcshilling** [comment]
#6278

**Comment by jimfb** [comment]
Yeah, I agree.  That's why I initially suggested posting in the react-native repository.  Doing something like https://github.com/facebook/react/pull/6278 would just hide the problem, making it more difficult to debug/reproduce/witness.  And even with that "fix", you would be loosing events, which is a bad user experience.  I'm going to close this out in favor of https://github.com/facebook/react-native/issues/5246, because I don't think there is anything actionable here.  If the RN team bounces it back here, we can re-investigate.

**Comment by everfire130** [comment]
I see the same crash in my release app.RN0.23.0.  iOS most in 9.3.1  . Has the issue been fixed in 0.24？

**Comment by marcshilling** [comment]
@everfire130 it's fixed in [0.24](https://github.com/facebook/react-native/releases/tag/v0.24.0). Scroll down to iOS bug fixes to see the detailed commits..."Fix how native events are coalesced when sending them to JS, specifically touch and scroll events, in order to respect their ordering"
